### PR TITLE
[WIP] Add Archlinux acceptance tests

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Comment by bastelfreak** [comment]
Depends on:
* https://github.com/voxpupuli/puppet-nginx/pull/1198
* https://github.com/voxpupuli/puppet-nginx/pull/1199
* https://github.com/voxpupuli/puppet-nginx/pull/1200

File: .travis.yml [hunk 1]
```diff
@@ -49,6 +49,18 @@ matrix:
...
+  - rvm: 2.4.3
+    bundler_args: --without development release
+    dist: trusty
+    env: PUPPET_INSTALL_TYPE=agent BEAKER_debug=true BEAKER_set=docker/debian-9 CHECK=beaker
+    services: docker
+    sudo: required
+  - rvm: 2.4.3
+    bundler_args: --without development release
+    dist: trusty
+    env: PUPPET_INSTALL_TYPE=foss BEAKER_debug=true BEAKER_set=docker/archlinux-4 CHECK=beaker
```
Comment (ekohl) [inline .travis.yml]: The ```type=foss``` needs to be an option in ```.sync.yml``` or the next modulesync will override it.

File: spec/acceptance/class_spec.rb [hunk 2]
```diff
@@ -52,9 +52,12 @@ class { 'nginx':
 
       describe package('nginx') do
         it { is_expected.to be_installed }
-        it 'comes from the expected source' do
-          pkg_output = shell(pkg_cmd)
-          expect(pkg_output.stdout).to match pkg_match
+        # the passenger people package ngin, except for Debian 9 systems!
```
Comment (ekohl) [inline spec/acceptance/class_spec.rb]: This is no longer needed because the other PR has a better fix.

**Comment by bastelfreak** [comment]
Hi people,
So this currently fails because of wrong defaults. beaker assumes the wrong paths for Puppet. They are based on these defaults:
https://github.com/puppetlabs/beaker-puppet/blob/b0c6608a7ea3060464807ce501060924b0fb7657/lib/beaker-puppet/install_utils/foss_defaults.rb#L73-L90

**Comment by bastelfreak** [comment]
Beaker is pretty broken for archlinux support, so I'm going to close this PR.

**Pull request closed without merging**